modules/mcagar/src/main/ts/ephox/mcagar/alien/EditorTypes.ts

```diff
@@ -1,5 +1,8 @@
 type EventCallback = (event: any) => void;
 
+export type GetContentFormatter = (editor: Editor, args: GetContentArgs) => string;
+export type SetContentFormatter = (editor: Editor, content: any, args: SetContentArgs) => string;
```

Comment:
The `content: any` is because we don't seem to expose AstNode and I didn't know how much of that we wanted to drag out. If anyone know a good solution please, do tell.